check_license/check-headers.sh

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
```

Comment:
when all done, the last thing to do would be to make order in your commits. In this PR it would be at best to have 3 commits:
1. adding check_license based on the pmemstream repository (with link in the description ,where was it copied from)
2. adding all your changes to the script
3. fixing copyrights and licenses